src/EditorFeatures/Core.Wpf/EditorFeatures.Wpf.csproj

```diff
@@ -2,6 +2,7 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <NoWarn>0618;1591</NoWarn>
```

Comment:
I needed all these NoWarns earlier when I was first adopting the 15.8 editor assemblies. Need to check if I can get rid of them now.